public/software_records.csv

```diff
@@ -0,0 +1,56 @@
+Software Record,Description,Status,Created on,Software Type,Vendor Record,Departments,Date Implemented,Date of upgrade,Developers,Tech Leads,Product Owners,Languages Used,Production URL,Source Code URL,User Seats,Annual Fees,Support Contract,Hosting Environment,Current Version,Notes,Business Value,IT Quality,Created By
```

Comment:
@hortongn Should this test data be in public or spec/fixtures?

Reply:
@scherztc I don't think this is test data.  I think these are .csv files that were generated by the export feature and got checked in to the repository by mistake.  Exporting the files to /public is fine, but we should update .gitignore to ignore any .csv files in that directory.

Reply:
That's a really good catch @scherztc and @hortongn. Those files are being generated by the export feature and I'm saving them locally on the project directory before a user can download them. I'll add them to .gitignore.